features/attr.yml

```
@@ -0,0 +1,19 @@
+name: attr
+description: The `attr()` CSS function returns the value of the specified attribute of an element. A default value may be provided if the attribute is missing or does not parse as the required type.
```

Being able to use `attr()` for things other than string values and in properties other than `content` has been a highly requested feature for years.

See https://issues.chromium.org/issues/40320391 and https://github.com/web-platform-tests/interop/issues/521 for example.

Because of this, I'd love it if we changed this feature's description to make it obvious to devs that the feature does cover this highly requested use case. Adding an example might do it:

```suggestion
description: The `attr()` CSS function returns the value of an HTML element's attribute, which can then be used as a typed value in a CSS declaration (types include length, number, angle, color, …). For example `width: attr(length);`. A default value may be provided if the attribute is missing or invalid.
```